### PR TITLE
Issue 1417 create set from characterisation result

### PR DESCRIPTION
**Pull request by AntonStepanof**
fixes https://github.com/OHDSI/Atlas/issues/1417

### CODE REVIEW

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js [hunk 1]
```diff
@@ -171,6 +184,58 @@ define([
             CsvUtils.saveAsCsv(exprt);
         }
 
+        createNewSet(analysis) {
+        	this.loading(true);
+        	const conceptIds = this.extractConceptIds(analysis);
+        	vocabularyProvider.getConceptsById(conceptIds)
+        		.then(({ data: items }) => { this.initConceptSet(items) })
```
Comment (pavgra) [inline js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: Please use async / await

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js [hunk 2]
```diff
@@ -171,6 +184,58 @@ define([
...
+        	});
+        	return conceptIds;
+        }
+
+        showConceptSet() {
+        	document.location = '#/conceptset/0/details';
+        }
+
+        initConceptSet(conceptSetItems) {
+        	return new Promise((resolve, reject) => {
```
Comment (pavgra) [inline js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: Please use async / await

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js [hunk 3]
```diff
@@ -171,6 +184,58 @@ define([
...
+        	analysis.data.forEach(r => {
+        		if (r.conceptId > 0) {
+        			conceptIds.push(r.conceptId);
+        		}
+        	});
+        	return conceptIds;
+        }
+
+        showConceptSet() {
+        	document.location = '#/conceptset/0/details';
```
Comment (pavgra) [inline js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: `commonUtils.routeTo`

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js (hunk 1 above)
Comment (AntonStepanof) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: updated

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js (hunk 2 above)
Comment (AntonStepanof) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: updated

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js (hunk 3 above)
Comment (AntonStepanof) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: updated

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js (hunk 1 above)
Comment (pavgra) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: Where is it "updated"? I see `.then`, `.catch` and `.finally`

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js (hunk 2 above)
Comment (pavgra) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: What _was_ updated? I see `Promise`

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js (hunk 1 above)
Comment (AntonStepanof) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: yes. fixed

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js (hunk 2 above)
Comment (AntonStepanof) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: yes. fixed

**Comment by chrisknoll** [comment]
@pavgra, please flush the toilet.

**Pull request merged**